tests/integration/clientside/assets/clientside.mjs

```diff
@@ -0,0 +1,5 @@
+import {display} from "./clientsideModule.mjs";
+
+window.dash_clientside.clientside_module = Object.assign({}, window.dash_clientside, {
+	display
+});
```

Comment:
Won't this put whatever else is already in `dash_clientside` into `dash_clientside.clientside_module`? ie:
```js
display = 5;
dash_clientside={something: {a: 1}};
window.dash_clientside.clientside_module = Object.assign({}, window.dash_clientside, {display});
console.log(JSON.stringify(dash_clientside,null,2));

{
  "something": {
    "a": 1
  },
  "clientside_module": {
    "something": {
      "a": 1
    },
    "display": 5
  }
}
```

Also just to ensure we're testing with the same patterns we recommend to users, the order of asset loading isn't guaranteed so each file that updates `dash_clientside` should be prepared to create it, something like:
```js
const dc = window.dash_clientside = (window.dash_clientside || {});
dc.clientside_module = {display};
// or if you imagine other modules contributing to clientside_module as well,
// use the same pattern again:
const cm = dc.clientside_module = (dc.clientside_module || {});
Object.assign(cm, {display});
```

Reply:
Good catch, yes! This was a copy-paste error and should be the same object that is read and assigned to, so:
```javascript
window.dash_clientside.clientside_module = Object.assign({}, window.dash_clientside.dash_clientside_module, {display});
```

The intention was to merge the (potentially) existing properties of `window.dash_clientside.dash_clientside_module` and  `{display}` to a new object (that is then assigned to `window.dash_clientside.clientside_module`).

I fixed it in the test, i found this to be the most concise way of writing it - but my javascript is a bit rusty.